### ISSUE TITLE
session not saved with web.FileResponse

### ISSUE THREAD

**Issue by Benj1** [aio-libs/aiohttp-session#640]
I'm not sure if there's a reason to exclude it, but it seems that sessions aren't saved when response is an instance of `web.FileResponse`.
It gets excluded here when checking the type of the response: https://github.com/aio-libs/aiohttp-session/blob/015dfb1acc035cf554252689bd00b9c944b8eab1/aiohttp_session/__init__.py#L204-L206

**Comment by Dreamsorcerer** [comment]
Have you tried removing the check and seeing if it works as expected?

**Comment by Benj1** [comment]
Yes, changing to `isinstance(response, (web.Response, web.HTTPException, web.FileResponse))` allows sessions to be set when returning static file responses

**Comment by asvetlov** [comment]
When I designed aiohttp-session, I made it work with non-streaming responses only.
Theoretically, the library can register `on_response_prepare` signal to dynamically update the response cookie if needed.
It can allow the processing of all responses, doesn't matter they are streamed or not.
Sorry, I have no idea when I can find time to work on the issue, any contribution is welcome!
